CMakeLists.txt

```diff
@@ -125,6 +125,23 @@ set(SCAP_WORKBENCH_LINK_LIBRARIES
     Qt5::Widgets Qt5::XmlPatterns
     ${OPENSCAP_LIBRARIES})
 
+# ---------- RPATHS for linking
+
+# see https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/RPATH-handling
+
+# when building, use the install RPATH
+# (but later on when installing)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+
+set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR})
```

Comment:
Doesn't `CMAKE_INSTALL_FULL_LIBDIR` resolve into a system path? Why add it to RPATH?